src/lib.rs

```diff
@@ -69,6 +70,16 @@ pub struct Server {
     static_directory: PathBuf,
 }
 
+impl fmt::Debug for Server {
```

Comment:
I added this in this PR because it was invaluable in debugging; we should have had it anyway. it needs a manual impl thanks to the closure.